nni/retiarii/codegen/pytorch.py

```diff
@@ -58,21 +59,25 @@ def _format_inputs(node: Node) -> List[str]:
                 inputs.append('{}[{}]'.format(edge.head.name, edge.head_slot))
     return inputs
 
-def graph_to_pytorch_model(graph_name: str, graph: Graph) -> str:
-    nodes = graph.nodes  # FIXME: topological sort is needed here
+def graph_to_pytorch_model(graph_name: str, graph: Graph, placement = None) -> str:
+    nodes = graph.topo_sort()  # FIXME: topological sort is needed here
 
     # handle module node and function node differently
     # only need to generate code for module here
     import_pkgs = set()
     node_codes = []
+    placement_codes = []
     for node in nodes:
         if node.operation:
             pkg_name = node.operation.get_import_pkg()
             if pkg_name is not None:
                 import_pkgs.add(pkg_name)
             node_code = node.operation.to_init_code(node.name)
             if node_code is not None:
-                node_codes.append(node_code)
+                if placement and node in placement and len(init_code) > 0:
```

Comment:
where is `init_code` initialized?

Reply:
I'm still working on fixing the hidden merging conflicts and bugs after syncing the upstream dev-retiarii branch.

Reply:
It should be node_code. Fixed.